src/creatures_panel.py

```diff
@@ -53,6 +53,7 @@ def draw_creature_panel(self, creature: Creature, context: ContextManager):
             f"Direction: {creature.direction.as_polar()[1]:.0f}° ({creature.direction.x:.3f}, {creature.direction.y:.3f})",
             f"Life: {creature.life} / {creature.max_life} (regen cost: {creature.life_regen_cost})",
             f"Energy: {creature.energy:.1f}",
+            f"Max Energy: {creature.max_energy:.1f}",
```

Comment:
For a better reading, I'd suggest to put the max energy value on the same line than the current energy value, ie. something like `Energy: 14 (max: 50)` or `Energy: 14/50`. More and more lines will be added as time goes by, if we can group the information now it's always a good thing :)